automation/create_workloads.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+set -ex
+
+CMD=oc
+
+for ns in {1..5}; do
+  ${CMD} create namespace "ns${ns}"
+  sed "s#__NS__#${ns}#"g automation/vm.yaml | ${CMD} apply -f -
+  ./virtctl start -n "ns${ns}" "testvm${ns}"
```

Comment:
you can define the VMs [here](https://github.com/kubevirt/must-gather/pull/84/files#diff-0b490f49df7ff9face7e12ba4004b5fc963316fb7a9754f4f0ffea591a4c0046R7) to be `running: true` and they will start up (and VMIs will be created) on definition, and you won't need to use `virtctl start`

Reply:
Maybe I'll use it in the future.